lib/DDG/Spice/BrainyQuote.pm

```diff
@@ -17,7 +17,8 @@ handle remainder => sub {
     }
     
     # Avoid triggering on 'stock' quotes; these are handled by Stocks IA
-    if ($req->query_lc =~ m/stock quote/) {
+    # Also avoid triggering on 'quote of the day' and 'quote for the day'; these are handled by QuoteOfTheDay IA
+    if ($req->query_lc =~ m/stock quote/ || $req->query_lc =~ m/quote of the day/ || $req->query_lc =~ m/quote for the day/ ) {
```

Comment:
Can we split this into ***two*** conditions? One with the first comment to prevent clashing with StockQuote, the other with the second comment to prevent clashing with QuoteOfTheDay.